### PR TITLE
Replace ◦ with `\circ` in math mode

### LINKED ISSUE #42: Fix \circ everywhere
Currently the symbol for `white bullet` `◦` is used everywhere instead of `ring operator` `∘`, because it won't render properly in PDF without using `\ensuremath`.

### PR DESCRIPTION
**Pull request by raol**
Closes #42

### CODE REVIEW

**Comment by Blaisorblade** [comment]
Thanks for this! I'd rather write

```tex
\newcommand \funcompose {\ensuremath{\circ}}
```
and use `\funcompose` for `\ensuremath{\circ}` to allow changing it if the need arises. The 2nd part might be change-and-replace.

Though I'd wait what @hmemcpy says—I'm just a passerby.

**Comment by hmemcpy** [comment]
I started replacing various `\code` with an inline `$f \circ b$`. It ends up looking "mathy" and no need to `\ensuremath` everywhere.

**Comment by Blaisorblade** [comment]
@hmemcpy That's more natural TeX; but multi-letter identifiers can be problematic.

Have you looked into lhs2TeX? That's what people use in many modern ICFP/Haskell Symposium papers—it lets you write `|foo . bar|` and convert it to `$\mathit{foo} \circ \mathit{bar}$` (where `\mathit` renders its argument as a word).

I've used lhs2TeX to write the actual math in my PhD thesis, which is an abuse I don't recommend; but using it for rendering Haskell-like code might be good. I suspect the category-theoretical formulas are "actual math" but lhs2TeX might scale for them.
*LaTeX offers lower-level control, which you need once the equations get complex enough.

Either way, if you are in for the long term, you might want to decide on lhs2TeX sooner rather than later.

**Comment by hmemcpy** [comment]
@Blaisorblade sounds great! I'll look into it, thanks!
Meanwhile, I started #56, where I attempt to use some macros I found... Any comments/improvements are most welcome!

**Comment by raol** [comment]
@hmemcpy as far as I understand `$ $` is the same as `\ensuremath{}`.  

Then I propose to close this PR, but I'll take part 2 and will convert it to math fully.
Ok?

**Comment by Blaisorblade** [comment]
> @hmemcpy as far as I understand $ $ is the same as \ensuremath{}.

Nitpick FYI: `\ensuremath` also works in math mode (which can be opened also by things that expand to `$` or `$$`), while `$...$` doesn't work when you're already inside math mode.

File: src/content/2.4/Representable Functors.tex [hunk 1]
```diff
@@ -245,7 +245,7 @@ \section{Representable Functors}\label{representable-functors}
 It must respect naturality conditions, and it must be the inverse of α:
 
 \begin{Verbatim}[commandchars=\\\{\}]
-α ◦ β = id = β ◦ α
+α  \ensuremath{\circ} β = id = β  \ensuremath{\circ} α
```
Comment (mseri) [inline src/content/2.4/Representable Functors.tex]: What about moving also `\alpha`, `\beta`, `\gamma`, `\epsilon`, ... to their LaTex math counterparts?

Comment (Blaisorblade) [reply src/content/2.4/Representable Functors.tex]: But the Unicode gives the same result (with the right packages like unicode-math, not sure it's setup here), and the source is more readable. Maybe we should move the Unicode alpha inside math mode.

Comment (mseri) [reply src/content/2.4/Representable Functors.tex]: Maybe this should be part of https://github.com/hmemcpy/milewski-ctfp-pdf/pull/56

Comment (hmemcpy) [reply src/content/2.4/Representable Functors.tex]: Yeah, I'm inclined to leave the unicode symbols as is, if they render fine. Most of them do.

**Comment by hmemcpy** [comment]
Yeah, I think to close this PR and, for now, just do the `$..$` etc. styling, as this wiki suggests https://en.wikibooks.org/wiki/LaTeX/Mathematics
Thanks again for the help!

**Pull request closed without merging**